bin/varnishd/cache/cache_conn_pool.c

```diff
@@ -218,6 +224,24 @@ VCP_AddRef(struct conn_pool *cp)
 	Lck_Unlock(&conn_pools_mtx);
 }
 
+/*--------------------------------------------------------------------
+ */
+
+static void
+vcp_free(struct conn_pool **cpp)
```

Comment:
I pointed out outside of github that `vcp_destroy()` was acting like a regular "free" function, but it was renamed to `vcp_free()` and is now behaving like a "destroy" function.

Apologies for the confusion. :sweat_smile:

Reply:
You did rename the function but forgot the call sites.